vuecomponents/pom.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.sakaiproject</groupId>
+        <artifactId>master</artifactId>
+        <version>23-SNAPSHOT</version>
+		<relativePath>../master/pom.xml</relativePath>
+    </parent>
+
+    <name>Sakai Vue Components</name>
+    <groupId>org.sakaiproject</groupId>
+    <artifactId>vuecomponents-parent</artifactId>
+    <packaging>pom</packaging>
+
+    <modules>
+        <module>bundle</module>
```

Comment:
Is this necessary?

Reply:
okay, we have a bundle for locales, thanks.

Reply:
Yes, for vuecomponets/bundle ;)